js/battle.js

```diff
@@ -553,7 +556,7 @@ var Pokemon = (function () {
 			if (plaintext) {
 				name += ' (' + this.species + ')';
 			} else {
-				name += ' <small>(' + this.species + ')</small>';
+				name = '<span class="pokemon" title="' +  this.species + '">' + name + ' <small>(' + this.species + ')</small>' + '</span>';
```

Comment:
Wouldn't removing that show the nicknames in tooltips etc. even with the option turned on? They're not part of the battle log so they wouldn't be affected by the CSS that hides it...